### PR TITLE
012_steeringWheelLocation_LEFT_2Apps_Apps_RAI_steeringWheelLocation

### PR DESCRIPTION
**Pull request by istoimenova**
Used common functions are implemented in
#603 
Script is part of TS: #597

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/RAI/040_ATF_P_steeringWheelLocation_LEFT_2Apps_Apps_RAI_steeringWheelLocation.lua [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+	event.matches = function(_, data) return data.method == "UI.GetCapabilities" end
+
+	EXPECT_HMIEVENT(event, "UI.GetCapabilities")
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, "UI.GetCapabilities", "SUCCESS", {
+			hmiCapabilities = 
+      {
+				navigation = false,
+				phoneCall = true,
+				steeringWheelLocation = "RIGHT"
```
Comment (aderiabin) [inline test_scripts/API/RAI/040_ATF_P_steeringWheelLocation_LEFT_2Apps_Apps_RAI_steeringWheelLocation.lua]: @istoimenova 
According test case description: In InitHMI_OnReady HMI replies with parameter steeringWeelLocation = LEFT to UI.GetCapabilities
In script it is steeringWheelLocation = "RIGHT"
Please check.

Comment (istoimenova) [inline test_scripts/API/RAI/040_ATF_P_steeringWheelLocation_LEFT_2Apps_Apps_RAI_steeringWheelLocation.lua]: @aderiabin, done in 849667e39dc9ea983e6a8b9d2f735e689defcf80

**Comment by istoimenova** [comment]
@aderiabin, please check updates in 849667e39dc9ea983e6a8b9d2f735e689defcf80

**Pull request merged**